### ISSUE TITLE
Auth readme needs to be rewritten

### ISSUE THREAD

**Issue by acoulton** [kohana/auth#22]
The auth module readme is basically some work-in-progress fork notes, and needs to be entirely rewritten. Skipping for the moment to get the build stuff running, but making a note to come back to it.

**Comment by shadowhand** [comment]
Personally, I think Auth module should be terminated with extreme prejudice... there are loads of better modules out there.

**Comment by acoulton** [comment]
@shadowhand indeed, but for as long as there's a 3.3 around it would be nice if it had a readme. I don't think I'd object to dropping it for 3.4 though.
